satellite/console/service.go

```diff
@@ -55,37 +56,36 @@ const (
 
 // Service is handling accounts related logic
 type Service struct {
-	Signer
+	log *zap.Logger
 
-	store DB
-	log   *zap.Logger
+	Signer
+	store         DB
+	stripeService payments.Service
 
 	passwordCost int
 }
 
 // NewService returns new instance of Service
-func NewService(log *zap.Logger, signer Signer, store DB, passwordCost int) (*Service, error) {
+func NewService(log *zap.Logger, signer Signer, store DB, stripeService payments.Service, passwordCost int) (*Service, error) {
```

Comment:
What happens if a particular Satellite Operator dont want to or can not use Stripe?
Can the satellite be operated without it?

Reply:
Honestly, I don't know. When we discussed payments we wanted to use stripe wherever we can, so decoupling it gonna break everything